### PR TITLE
Replacing refs to 'master' branch with 'main'

### PR DESCRIPTION
**Pull request by bocon13**
Also requires renaming 'master' to 'main' on the p4lang repo and updating the default branch in GitHub Settings:
https://github.com/github/renaming#rename-existing

### CODE REVIEW

**Comment by ccascone** [comment]
Agreed to rename the default branch. However, it looks like Github is still working on making the transition seamless for existing repositories: https://github.com/github/renaming/#later-this-year-seamless-move-for-existing-repositories-

We should consider waiting to avoid disruption to other projects depending on this repo (e.g., scripts explicitly referring to `master` for git operations)

File: docs/v1/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ documentation are available at **[TODO]**
 Files:
 - `P4Runtime-spec.mdk` is the main file. 
 - assets: Figures
-  - `*.odg` - OfficeLibre master drawing file used to export images. These are
+  - `*.odg` - OfficeLibre drawing file used to export images. These are
```
Comment (bocon13) [inline docs/v1/README.md:16]:
```suggestion
  - `*.odg` - OfficeLibre authoritative drawing file used to export images. These are
```

**Comment by bocon13** [comment]
Yes, I think it makes sense to wait until GitHub sets up branch (git & http) redirection if possible

**Comment by bocon13** [comment]
It looks like Github now has support for branch renaming:
https://github.com/github/renaming#rename-existing

It updates existing PRs, supports HTTP redirects, and provides a notice to users of the git CLI.

File: README.md [hunk 2]
```diff
@@ -21,12 +21,12 @@ Git tags are used to mark minor and patch release versions.
 ## Reading the latest version of the documentation
 
 The latest version of the P4Runtime v1 specification is available:
-* [here](https://p4.org/p4runtime/spec/master/P4Runtime-Spec.html) in **HTML**
+* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.html) in **HTML**
   format
-* [here](https://p4.org/p4runtime/spec/master/P4Runtime-Spec.pdf) in **PDF**
+* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.pdf) in **PDF**
```
Comment (antoninbas) [inline README.md:26]: These links are referenced in p4.org, but since you mention there is automatic redirection, it should not be an issue

**Review by antoninbas — APPROVED** [review]
LGTM

**Pull request merged**